src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageReadmeControl.xaml.cs

```diff
@@ -17,54 +20,103 @@ namespace NuGet.PackageManagement.UI
     /// <summary>
     /// Interaction logic for PackageReadmeControl.xaml
     /// </summary>
-    public partial class PackageReadmeControl : UserControl
+    public partial class PackageReadmeControl : UserControl, IDisposable, INotifyPropertyChanged
     {
 #pragma warning disable CS0618 // Type or member is obsolete
         private IMarkdownPreview _markdownPreview;
 #pragma warning restore CS0618 // Type or member is obsolete
+        private bool _isDisposed = false;
+        private CancellationTokenSource _cancellationTokenSource;
```

Comment:
The type already tells us it's a cancellation token source. It should have a name that describes it's usage, like `_isDisposed` is not named `_bool`.

Perhaps something like `_controlDisposingTokenSource`